packages/block-editor/src/components/block-navigation/list.js

```diff
@@ -59,7 +59,7 @@ export default function BlockNavigationList( {
 		 */
 		/* eslint-disable jsx-a11y/no-redundant-roles */
 		<ul className="editor-block-navigation__list block-editor-block-navigation__list" role="list">
-			{ map( blocks, ( block ) => {
+			{ map( omitBy( blocks, isNil ), ( block ) => {
```

Comment:
Nice catch! I have a newbie question: How does a `nil` value end up in `blocks` array? Is there a use-case for it?

Reply:
I'm not sure there is a use case for it and it probably shouldn't end up there. That's something we should look at separately. This PR mainly addresses the fact that it did end up here and we should thus safeguard against that happening.